docs/guide/configuration/devices-groups.md

```diff
@@ -98,6 +98,15 @@ and `debounce` option without `debounce_ignore` publishes only last payload with
 hand `debounce: 1` with `debounce_ignore: - action` will publish all unique action messages, at least two (
 e.g. `action: rotate_left` and `action: rotate_stop`)
 
+**`throttle`**  
+Throttle messages of this device. When setting e.g. `throttle: 10` the first message from device is accepted but all other messages in next 10 seconds will be ignored. Be carefull when using this option, unlike debounce that can ignore some attributes this will drop entire message.
```

Comment:
```markdown
Throttle processing of messages from this device. When setting e.g. `throttle: 10` the first message from the device is processed but all other messages within the next 10 seconds are ignored. Be careful when using this option, unlike `debounce` that can ignore some attributes, this will drop the entire message.
```